Makefile

```diff
@@ -119,19 +113,13 @@ verify-operator-bundle: operator-sdk
 	hack/verify-operator-bundle.sh
 
 operator-bundle:
-	@echo "FUSION=$(FUSION)"
```

Comment:
Same, This change is not required.

Reply:
these changes occured after, make gen-csv, as mentioned in earlier CI failures. Cool I will rebase it.